components/Cards/index.js

```diff
@@ -17,3 +17,52 @@
 // </div>
 //
 // Create a card for each of the articles and add the card to the DOM.
+
+
+axios.get("https://lambda-times-backend.herokuapp.com/articles")
+    .then(response => {
+        console.log("Got data from Lambda");
+        console.log(response);
+        console.log(response.data.articles) // logging article topics - each has multiple articles
```

Comment:
how would you loop through this data in order to individually feed it into the article element creator function?